### PR TITLE
Bump SmallRye Stork to 1.0.0

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: independent-projects/resteasy-reactive/pom.xml [hunk 1]
```diff
@@ -314,7 +314,7 @@
             </dependency>
             <dependency>
                 <groupId>io.smallrye.stork</groupId>
-                <artifactId>smallrye-stork-api</artifactId>
+                <artifactId>stork-core</artifactId>
```
Comment (cescoffier) [inline independent-projects/resteasy-reactive/pom.xml:317]: Shouldn't it depend on the bom?

**Review by cescoffier — CHANGES_REQUESTED** [review]
Just one small comment.

File: independent-projects/resteasy-reactive/pom.xml (hunk 1 above)
Comment (gsmet) [reply independent-projects/resteasy-reactive/pom.xml:317]: Yeah, it would be better to import the BOM here too.

Comment (michalszynkiewicz) [reply independent-projects/resteasy-reactive/pom.xml:317]: I wasn't sure about it. It's an independent project. I can change to Bom.

Comment (michalszynkiewicz) [reply independent-projects/resteasy-reactive/pom.xml]: changed to bom import

**Comment by michalszynkiewicz** [comment]
`MultiModuleNamedInjectionDevModeTest` failed for Gradle JDK 11 tests

**Review by cescoffier — APPROVED** [review]

**Comment by gsmet** [comment]
@michalszynkiewicz yeah don't worry about that. Let's wait for the full build result and we will see the report.

**Pull request merged**